### ISSUE TITLE
 Unable to resolve module AccessibilityInfo from /Users/admin/ebookDemo/node_modules/epubjs-rn/node_modules/react-native/Libraries/react-native/react-native-implementation.js: AccessibilityInfo could not be found within the project or in these directories:

### ISSUE THREAD

**Issue by chandan-mishra-iphtech** [GeekyAnts/NativeBase-KitchenSink#210]

**Comment by chandan-mishra-iphtech** [comment]
after installing epub file in my project i got this error
